drivers/scheduler/k8s/specs/kubevirt-multi-vm/vm-ubuntu-data-volume.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-ubuntu-data-volume
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: boot-volume
+          - disk:
+              bus: virtio
+            name: cloud-initdisk
+          interfaces:
+          - masquerade: {}
+            name: default
+        resources:
```

Comment:
Also add machine: type: q35 for intel bases system. Else this will cause version mismatch on OCP between older and newer.

Reply:
As of now I don't see such issues.. Can you give some more info on this.